airflow/bin/cli.py

```diff
@@ -325,6 +325,21 @@ def task_state(args):
     print(ti.current_state())
 
 
+def dag_state(args):
+    """
+    Returns the state of a DagRun at the command line.
+
+    >>> airflow dag_state tutorial 2015-01-01
```

Comment:
Can you provide an example where the execution date is more than daily (e.g. hourly)?